deployment-logs/stagelight.log

```diff
@@ -0,0 +1,3 @@
+- v1.1.0 deployed on 2025-06-03
```

Comment:
Can we also add a commit sha, that has been deployed, also maybe the github-ID of the guy who deployed it.

Reply:
I'll add commit and deployer 100%.
About the logs, I feel it can bee too much. WDYT @NindoK?

Reply:
Actually, I think the commit will fit better in the Changelog. Since it's there where we define versions.

Reply:
About logs, guys. Depend on how much it is, but I still think it might be really helpful.